src/vs/workbench/contrib/outline/browser/outlinePane.ts

```diff
@@ -571,7 +571,8 @@ export class OutlinePane extends ViewPane {
 			// todo@Joh
 			if (e.browserEvent) {
 				if (e.browserEvent.type === 'keydown') {
-					focus = true;
+					const selectionEvent = e.browserEvent as SelectionKeyboardEvent;
+					focus = typeof selectionEvent.preserveFocus === 'boolean' ? selectionEvent.preserveFocus : true;
```

Comment:
```suggestion
					focus = typeof selectionEvent.preserveFocus === 'boolean' ? !selectionEvent.preserveFocus : true;
```

The boolean logic is flawed here I think, can you please test this.

Reply:
We don't really need this check, the old code was using it, and I forgot to remove this code while rebasing.